.gitignore

```diff
@@ -4,3 +4,4 @@
 /.idea
 /node_modules/
 /site/
+Pipfile
```

Comment:
Why did you add this here?

Reply:
It was used for my local Python env but not intended to merge, to be honest I forgot remove after my last commit. Removed from current PR.

Reply:
Images compressed and resized to a smallest size.